README.md

```diff
@@ -76,3 +76,11 @@ Run the following commands:
 cd /usr/share/cephmetrics-ansible
 ansible-playbook -v -i ~/cephmetrics/inventory -e '@~/cephmetrics/vars.yml' playbook.yml
 ```
+## Known Issues
+1. The dashboard queries rely on a template variable called domain which provides the dns
+domain name for the ceph hosts. If your environment is not using dns, the collectd data
+sent to graphite will not include a dns suffix in the metric name. Without the dns suffix
+the dashboard queries fail, and your dashboard panels show N/A.  
+ **Workaround**: On each of your ceph nodes, update collectd.conf to use a fully qualified 
+  domain name on the HostName parameter. This will ensure the domain name component
```

Comment:
I have another way that this can be resolved.We specify the environment in the dashboard.yml - which declares the domain to use in all the dashboards. I've tested a minor patch to dashUpdater that basically removes references to $domain from the dashboard json files if the domain is not set in dashboard.yml. This way there isn't any change to collectd on the hosts or concerns over reruns. If that sounds OK, can you approve this PR and I'll raise another to fix up the readme and add the functionality to dashupdater